src/test/TestHelper.cpp

```diff
@@ -22,15 +22,23 @@ void SetFileName(std::string& file, std::string name)
 
 bool adaptive::AdaptiveTree::download(const char* url,
                                       const std::map<std::string, std::string>& manifestHeaders,
+                                      const std::string &manifestUpdateParam,
                                       void* opaque,
-                                      bool scanEffectiveURL)
+                                      bool isManifest)
 {
   FILE* f = fopen(testHelper::testFile.c_str(), "rb");
   if (!f)
     return false;
 
-  if (scanEffectiveURL && !testHelper::effectiveUrl.empty())
-    SetEffectiveURL(testHelper::effectiveUrl);
+  effective_url_ = url;
+  if (!testHelper::effectiveUrl.empty())
```

Comment:
Rather than assign twice here a if/else would be clearer

Reply:
My reasoning for this was to keep it as close as the real function. This line is where the effective url would be retrieved from curl. 

We need keep the real and mocked function as close as possible to keep the tests accurate?